.eslintrc.ts.js

```diff
@@ -0,0 +1,18 @@
+// eslint config for .ts / .d.ts files
```

Comment:
Consider combining both `.eslintrc.js` and `.eslintrc.ts.js` into a single file using [`overrides`](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns).